lib/service_extensions.dart

```diff
@@ -0,0 +1,7 @@
+const String debugPaintExtName = 'ext.flutter.debugPaint';
```

Comment:
Add a library name.

Note that some of our files are missing copyright notices. at some point soon we need to go through and add them everywhere. No rush.

Reply:
Done.